OpenRA.Mods.RA/Effects/SatelliteLaunch.cs

```diff
@@ -43,7 +43,7 @@ public void Tick(World world)
 			if (++frame == 19)
 			{
 				var palette = info.SatellitePaletteIsPlayerPalette ? info.SatellitePalette + launcher.Owner.InternalName : info.SatellitePalette;
-				world.AddFrameEndTask(w => w.Add(new GpsSatellite(world, pos, info.SatelliteImage, info.SatelliteSequence, palette)));
+				world.AddFrameEndTask(w => w.Add(new GpsSatellite(world, pos, info.SatelliteImage, info.SatelliteSequence, palette, info.RevealDelay * 25, launcher.Owner)));
```

Comment:
Basically I agree and was thinking about doing that, but in my opinion it would be better to do *all* support power "seconds > ticks" conversions in one dedicated PR, I don't like the thought of having both cases mixed on bleed.